### DISCUSSION TITLE
Can `Genie` web app be converted into a desktop app by `tauri` or `electron`?

### DISCUSSION THREAD

**Discussion by zkk960317** [GenieFramework/Genie.jl#710] [General]
Can `Genie` web app be converted into a desktop app by `tauri` or `electron`? Any example?

**Comment by essenciary** [comment]
Yes, it should work as there is a contributed demo. We're planning on taking a closer look at this and releasing some docs soon.

**Comment by PGimenez** [comment]
We're working on a new package here https://github.com/GenieFramework/StippleDesktop.jl

**Comment by Rex-cy** [comment]
yes we can make it
